api/http.go

```diff
@@ -66,9 +67,13 @@ func NewCatalystAPIRouter(cli config.Cli, vodEngine *pipeline.Coordinator, bal b
 
 	router.GET("/ok", withLogging(catalystApiHandlers.Ok()))
 	if cli.EnableAnalytics == "true" || cli.EnableAnalytics == "enabled" {
-		analyticsApiHandlers := handlers.NewAnalyticsHandlersCollection(mapic, lapi, cli.AnalyticsMetricsURL, cli.NodeName)
-		router.POST("/analytics/log", withLogging(withCORS(analyticsApiHandlers.Log())))
-		router.GET("/analytics/log", withLogging(withCORS(geoHandlers.RedirectHandler())))
+		analyticsApiHandlers, err := handlers.NewAnalyticsHandlersCollection(mapic, lapi, cli.KafkaBootstrapServers, cli.KafkaUser, cli.KafkaPassword, cli.KafkaTopic)
+		if err != nil {
+			glog.Errorf("failed to configure analytics handlers, analytics endpoint won't be enabled, err=%v", err)
```

Comment:
I believe startup should fail if we get an error here. Better to block a deploy than never realize we broke our data pipeline.

Reply:
ok, changed to `Fatalf()` which will stop catalyst-api.